### DISCUSSION TITLE
How to fetch next runtime and last runtime using this package?

### DISCUSSION THREAD

**Discussion by achyutghosh** [dahall/TaskScheduler#937] [Q&A] [answered]
I've used this package to create the scheduler. But now I need to fetch the next runtime of the scheduler. Can anyone suggest to me any code snippet to fetch the next retry time or last retry time?

**Comment by dahall** [comment, marked as answer]
Does #801 help?

**Reply by achyutghosh** [reply]
Thnx, this was helpfull
